plutus-core/generators/Language/PlutusCore/Generators/NEAT/Type.agda

```diff
@@ -0,0 +1,327 @@
+-- This file is the source Agda file
+-- Edit this file not Type.hs
+-- The warning below will be written to Type.hs
+
+module Language.PlutusCore.Generators.NEAT.Type where
+
+-- warning to be written to Haskell file:
+{-# FOREIGN AGDA2HS
+{-
+!!! THIS FILE IS GENERATED FROM Type.agda
+!!! DO NOT EDIT THIS FILE. EDIT Type.agda
+!!! AND THEN RUN agda2hs ON IT.
+-}
+#-}
+
+{-# FOREIGN AGDA2HS
```

Comment:
How does this parse??

Reply:
oops, missed that. I would have to defer to Ulf for that